src/SDKs/Azure.Core/data-plane/Azure.Core/Pipeline/Policies/ClientRequestIdPolicy.cs

```diff
@@ -19,7 +19,7 @@ protected ClientRequestIdPolicy()
 
         public override Task ProcessAsync(HttpPipelineMessage message, ReadOnlyMemory<HttpPipelinePolicy> pipeline)
```

Comment:
Similar to the thought that I mentioned in `HttpHeader`,  because these are more Azure-specific headers and not the more frequently used request identifier headers, maybe we should consider renaming this to `AzureClientRequestIdPolicy` or similar?

Reply:
Not sure if we should, we are not trying to produce a general-purpose client, Azure is in the namespace.